boards/shields/zest_display_lcd/zest_display_lcd.overlay

```diff
@@ -2,3 +2,46 @@
  * Copyright (c) 2024, CATIE
  * SPDX-License-Identifier: Apache-2.0
  */
+
+#include <zephyr/dt-bindings/gpio/sixtron-header.h>
+#include <zephyr/dt-bindings/display/panel.h>
+
+/ {
+	chosen {
+		zephyr,display = &ili9163c0;
+	};
+
+	pwm_devices {
+		status = "okay";
+		compatible = "pwm-leds";
+
+		backlight_lcd: pwm-dev {
+			status = "okay";
+			pwms = <&pwm1 3 2000000 PWM_POLARITY_NORMAL>;
+		};
+	};
+
+	mipi_dbi {
+		compatible = "zephyr,mipi-dbi-spi";
+		dc-gpios = <&sixtron_connector DIO2 GPIO_ACTIVE_HIGH>;  /* Data/Command */
+		spi-dev = <&sixtron_spi>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		write-only;
+
+		ili9163c: ili9163c@0 {
+			compatible = "ilitek,ili9163c";
+			mipi-max-frequency = <20000000>;  // 20MHz
```

Comment:
Comments should follow coding style (see here: https://docs.kernel.org/6.8/devicetree/bindings/dts-coding-style.html).